packages/ui/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@prezly/theme-kit-ui",
+  "version": "0.0.1",
+  "description": "UI components for Prezly themes",
+  "keywords": ["prezly", "themes", "ui"],
+  "author": "Prezly Team",
+  "homepage": "https://github.com/prezly/theme-kit-js/tree/main/packages/ui#readme",
+  "license": "MIT",
+  "main": "build/index.js",
+  "files": ["build", "README.md"],
+  "types": "build/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/prezly/theme-kit-js.git"
+  },
+  "scripts": {
+    "prebuild": "rimraf build/*",
+    "build": "microbundle build --tsconfig ./tsconfig.json --compress --format modern,cjs src/index.ts",
+    "dev": "microbundle watch --tsconfig ./tsconfig.json --format modern,cjs src/index.ts",
+    "build-storybook": "storybook build",
+    "storybook": "storybook dev -p 6006"
```

Comment:
Please make sure to set up Storybook deployment in Chromatic, this will make reviews much easier.
Let me know if you need help with the set up